wvd-sh/WVD scaling script/basicScale-MSI.ps1

```diff
@@ -742,19 +780,26 @@ else {
                 foreach ($sessionHost in $RDSessionHost.sessionhostname) {
                     if ($hostPoolUserSessions.Count -ge $AvailableSessionCapacity) {
                         $VMName = $sessionHost.Split(".")[0]
+                          $VmInfo = Get-AzureRmVM | Where-Object { $_.Name -eq $VMName }
+                            #Check the Session host is in Maintainance
+                            if($VmInfo.Tags.Keys -contains $maintainanceTagName){
+                                 Write-Log 1 "Session Host is in Maintainace: $($sessionhost | out-string)"
```

Comment:
Replace "Maintainace" with "maintenance"

Reply:
We have replaced in both scripts